include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -1245,6 +1245,8 @@ namespace detail
              *    is opt-in for now.
              * 2) Reading with the Streaming API any file that has been written
              *    without steps.
+             *
+             * @todo [franzpoeschel] let's remove these work-arounds :-)
```

Comment:
The description in this one is a bit incomplete. As of ADIOS 2.7.0, it is still necessary to delay opening the first step in case a reader does not use steps at all.
But in any case, I'll need to update the description of this (and also mention somewhere that its difference to the state `OutsideOfStep` lies in the fact that `requireActiveStep()` will not open a step).